### DISCUSSION TITLE
file yml

### DISCUSSION THREAD

**Discussion by ghost** [projectdiscovery/nuclei-templates#8762] [Q&A]
Hi I would like ask if file yml is the same as Nuclei use .I can use this in Nuclei .It will be works .???

https://raw.githubusercontent.com/huntresslabs/threat-intel/main/2023/2023-11/suspicious_child_process_from_activemq.yml

**Comment by MetzinAround** [comment]
No @kolor33 , this would not run in Nuclei as it doesn't have the yaml key values that are required for Nuclei. 

You can use it with AI to try and make a correct template here: cloud.projectdiscovery.io. Use the AI function and input that yaml, then go from there. It should help get you started in building this template.
